packages/agoric-cli/lib/deploy.js

```diff
@@ -75,11 +75,11 @@ export default async function deployMain(progname, rawArgs, powers, opts) {
             { externals: builtinModules },
           );
 
+          const nestedEvaluate = src =>
+            evaluateProgram(src, { require, HandledPromise, nestedEvaluate });
```

Comment:
Do you make use of the recursion here, that `src` is endowed with `nestedEvaluate` as well?

Reply:
Yes, that's the nature of `nestedEvaluate`; it evaluates a script which in turn evaluates sub-scripts.